### DISCUSSION TITLE
Weird issue color presets, some of the colors make text disappear?

### DISCUSSION THREAD

**Discussion by oscklm** [tailwindlabs/tailwindcss#11380] [Help] [answered]
So i'm having this weird issue all of a sudden, where random colors from the built in`text-[color]-[100-900]` just wont show a color and the text disappears.

Kind of bad gif quality, but hope it's possible to tell from it:
![CleanShot 2023-06-06 at 12 51 38](https://github.com/tailwindlabs/tailwindcss/assets/22825865/96a079c9-de23-4557-a420-bd1fe2b642a9)

```js
// tailwind.config.js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    './pages/**/*.{js,ts,jsx,tsx}',
    './components/**/*.{js,ts,jsx,tsx}',
    './app/**/*.{js,ts,jsx,tsx}',
  ],
  theme: {
    extend: {
      fontFamily: {
        default: ['var(--font-inter)'],
      },
    },
  },
  plugins: [],
};

```

I've never experienced anything like this before. Super odd.

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you changed the class.
- The `content` file globs sufficiently cover the file that the code appears in.

Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by oscklm** [reply, marked as answer]
I was actually just blinded by the most obvious thing. The text-transparent left over from a previous way i styled that same text, and for some reason i didn't think of that being the issue, since the color classes did work for some of the values from 100-900.

It's fixed now. And i'll mark as answered 👍
